README.md

```diff
@@ -87,10 +87,11 @@ service:
 
 | parameter    | description                                           | required | default                  |
 | ------------ | ----------------------------------------------------- | -------- | ------------------------ |
+| accessToken  | ArgoCD Access Token                                   | `false`  |                          |
```

Comment:
Add this to `action.yml`: https://github.com/ejhayes/action-argocd-deploy/blob/next/action.yml#L4

This readme is updated automatically based on that file